src/appMain/main.cc

```diff
@@ -55,6 +56,10 @@ int main(int argc, char* argv[]) {
 
   INIT_LOGGER(profile_instance.logs_enabled());
   CREATE_LOGGERPTR_LOCAL(logger_, "SDLMain")
+  if (!utils::appenders_loader.Loaded()) {
+     LOG4CXX_ERROR(logger_,
```

Comment:
Maybe `DCHECK`?
Seems in case of buildign with logs and setting `true` in config file working startign with appenders is wrong at all and could be `DCHECked`.